v2/app/testdata/pubsub_err_subscriber_different_service.txt

```diff
@@ -1,5 +1,5 @@
 ! parse
-err 'The function passed to `pubsub.NewSubscription` must be declared in the the same service as the'
+err 'The function passed to `pubsub.NewSubscription` must be declared in the same service as the'
```

Comment:
This is testing that the parser returns a specific error. You'll need to update the code that returns that error, too, otherwise the test will fail